charter/index.html

```diff
@@ -28,269 +28,278 @@
+      experience significantly. A new generation of VR headsets are emerging
+      that integrate depth sensing into their products (e.g. see the Intel
+      product announced at CES 2016). Another spec in the CG will produce an
+      experimental Web Hand API to allow users to see their hands in the
+      virtual environment.
+    </p>
+    <p>
+      The APIs use the web’s security model based on origins, and as such
+      target products are web browsers. Specific security and privacy
+      considerations constrain the scope of the specification to make the API
+      implementable in web browsers without compromising the user’s security or
+      privacy. Specific security considerations for the WebVR API are
+      enumerated in the specification.
+    </p>
+    <h3 id="out-of-scope">
+      Out of Scope
+    </h3>
+    <p>
+      The group will not define the underlying VR system that the Web APIs use.
+      These APIs give access to capabilities already provided on systems. The
+      group will not define media codecs.
+    </p>
     <h2 id="deliverables">
       Deliverables
     </h2>
-    
-      <h3 id="specifications">
-        Specifications
-      </h3>
-    
-      <p class="remove">
-        {TBD: Provide a brief description of each specification the group plans 
-        to produce. Where an estimate is possible, it can be useful to provide
-        an estimated schedule for key deliverables. As described below, 
-        the group may later modify the charter deliverables. if no specifications, 
-        include: "No Specifications will be produced under the current charter."} 
-      </p>
-
-      <h3 id="non-normative-reports">
-        Non-Normative Reports
-      </h3>
-    
-      <p>The group may produce other Community Group Reports within
-      the scope of this charter but that are not Specifications, for instance use cases, 
-      requirements, or white papers.</p>   
-
-      <h3 id="test-suites">
-        Test Suites and Other Software
-      </h3>
-    
-      <p class="remove">{TBD: State whether test suites or any other software 
-       will be created for any Specifications and list the relevant licenses. 
-       For information about contributions to W3C test suites, please see 
-       <a href="http://testthewebforward.org/">Test the Web Forward</a>,
-       and take note of the <a href="http://www.w3.org/2004/10/27-testcases">W3C's
-       test suite contribution policy</a> and 
-       <a href="http://www.w3.org/Consortium/Legal/2008/04-testsuite-copyright.html">licenses
-       for W3C test suites></a>. If there are no plans to create a test suite 
-       or other software, please state that.}</p> 
-    
+    <h3 id="specifications">
+      Specifications
+    </h3>
+    <p>
+      <a href="http://mozvr.github.io/webvr-spec/">WebVR API</a>: API for
+      accessing Virtual Reality (VR) devices, starting from Web pages Incubator
```

Comment:
I think this should say `API for accessing VR devices from Web pages (Incubator Draft spec).`

The "Incubator Draft spec" referring to this spec to be used as input for incubation in this CG. I'll land a fix to this separately.